templates/back-office/gift-receipt-content.hypr

```diff
@@ -0,0 +1,102 @@
+{% comment %} 
+    Model passed here is Shipment object. It will have all updated gift messages.
+    Since pickup location address is not present on Shipment, it is passed separately from storefront in ViewData.
+{% endcomment %}
+<table class="mz-printable-order gift-receipt">
+    <thead class="header">
+        <tr>
+            <th>
+                <h3>Gift Receipt</h3>
+            </th>
+            <th class="order-essentials">
+                <div class="order-number"><span class="label">Order</span><span>#{{ model.orderNumber }}</span></div>
```

Comment:
Why we are not localizing the labels here like `Order` , `External Order` ?

Reply:
Done